components/landing/banner/Banner.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styles from './Banner.module.scss';
+
+import Testimonial from '../../../components/testimonial/Testimonial';
```

Comment:
this import should look like this.. 
`import { Testimonial } from '../../testimonial';
`

In order for the above import to work we will have to make changes to the components/testimonial file structure, I will leave comments below for that